pkg/lib/cockpit-po-plugin.js

```diff
@@ -105,7 +103,17 @@ function init(options) {
 }
 
 function run(webpack_module, webpack_compilation) {
-    return Promise.all(get_po_files().map(f => buildFile(f, webpack_module, webpack_compilation)));
+    return Promise.all(get_po_files().map(f => {
+        if (webpack_compilation)
+            webpack_compilation.fileDependencies.add(f);
+        const lang = path.basename(f).slice(0, -3);
+        return Promise.all([
+            buildFile(f, webpack_module, webpack_compilation,
+                      `${config.subdir}po.${lang}.js`, str => !str.includes('manifest.json') && !str.includes('manifest.json.in')),
```

Comment:
The .in variants are redundant. I only brought that up as a possible scenario why .includes() is more correct than .endsWith(). Please drop them again.

Reply:
*facepalm* yes ofcourse that makes sense.